### ISSUE TITLE
Warning: Each child in a list should have a unique "key" prop.

### ISSUE THREAD

**Issue by andruxnet** [cawfree/react-native-twitter-textview#1]
Hi,

I have a fix for the warning above, how can I send you a PR?

**Comment by cawfree** [comment]
Awesome, thanks a lot! 

If you `clone` the repo, make your changes and bump the `package.json` version, head over to [Pull Requests](https://github.com/cawfree/react-native-twitter-textview/pulls) and choose to merge your copy of `master` into `cawfree/master`.

Thanks again!

**Comment by andruxnet** [comment]
It doesn't seem like I have access to the repository..

**Comment by cawfree** [comment]
Hey @andruxnet , sorry for the delay getting back to you. Are you saying you can't get to [this](https://github.com/cawfree/react-native-twitter-textview/compare) url?

**Comment by andruxnet** [comment]
I can, but I can’t push to your repo

**Comment by Asinox** [comment]
Please... PR

Thanks!

**Comment by andruxnet** [comment]
I can't, when I try to push my fix branch I get this:

```ERROR: Permission to cawfree/react-native-twitter-textview.git denied to andruxnet.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.```

**Comment by cawfree** [comment]
Hey, since it looks like you’re still having trouble I will port over the changes this evening, alongside a couple of enhancements. 😅

**Comment by cawfree** [comment]
Fixed in [1.0.4](https://github.com/cawfree/react-native-twitter-textview/commit/a0d8fea37ca2b275619a853225abc42005cb75eb). Thanks!
